### PR TITLE
[Serializer] Add feature description for `AbstractNormalizer::REQUIRE_ALL_PROPERTIES` context option

### PR DESCRIPTION
**Pull request by christian-kolb**
This PR adds the feature description for the new context flag for the Serializer component introduced here: https://github.com/symfony/symfony/pull/49553

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -1242,6 +1242,31 @@ to ``true``::
...
+
+    $data = ['foo' => 'notNull'];
+
+    $normalizer = new ObjectNormalizer();
+    $result = $normalizer->denormalize($data, Dummy::class, 'json', [AbstractNormalizer::PREVENT_NULLABLE_FALLBACK => true]);
+    // throws Symfony\Component\Serializer\Exception\MissingConstructorArgumentException
+
+.. versionadded:: 6.3
+
+    The context flag was introduced in Symfony 6.3.
```
Comment (OskarStark) [inline components/serializer.rst]:
```suggestion
    The ``AbstractNormalizer::PREVENT_NULLABLE_FALLBACK`` context option
    was introduced in Symfony 6.3.
```

File: components/serializer.rst [hunk 2]
```diff
@@ -1242,6 +1242,31 @@ to ``true``::
     $result = $normalizer->normalize($dummy, 'json', [AbstractObjectNormalizer::SKIP_NULL_VALUES => true]);
     // ['bar' => 'notNull']
 
+Preventing ``null`` value fallback for nullable properties
```
Comment (OskarStark) [inline components/serializer.rst]:
```suggestion
Preventing ``null`` Value Fallback for Nullable Properties
```

File: components/serializer.rst [hunk 3]
```diff
@@ -1242,6 +1242,31 @@ to ``true``::
...
+----------------------------------------------------------
+
+By default, the Serializer will add `null` to nullable properties when the parameters for those are not provided.
```
Comment (OskarStark) [inline components/serializer.rst]:
```suggestion
By default, the Serializer will add ``null`` to nullable properties when the parameters for those are not provided.
```

File: components/serializer.rst [hunk 4]
```diff
@@ -1242,6 +1242,31 @@ to ``true``::
...
+You can change this behavior by setting the ``AbstractNormalizer::PREVENT_NULLABLE_FALLBACK`` context option
+to ``true``::
+
+    class Dummy {
```
Comment (OskarStark) [inline components/serializer.rst]:
```suggestion
    class Dummy
    {
```

File: components/serializer.rst (hunk 2 above)
Comment (christian-kolb) [reply components/serializer.rst]: @OskarStark Aren't all nouns written in lowercase in English (except countries and languages)? Or is this a convention of the Symfony documentation?

Comment (OskarStark) [reply components/serializer.rst]: For reference:
https://symfony.com/doc/current/contributing/documentation/standards.html#english-language-standards

Comment (christian-kolb) [reply components/serializer.rst]: @OskarStark Thanks for the reference, I've committed the changes 🙂

Comment (OskarStark) [reply components/serializer.rst]: Thank you!

**Review by OskarStark — APPROVED** [review]

**Review by maxbeckers — APPROVED** [review]

**Comment by christian-kolb** [comment]
@OskarStark The connected PR was just merged 🙂
Can this also be merged? I saw that there have been a lot of commits in the meantime. Otherwise I do the changes again on the current version if that helps.

**Comment by xabbuh** [comment]
Thank you @christian-kolb.

**Pull request merged**